### ISSUE TITLE
Mobile Signal Work Orders

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-data-tech#251]
## Description
We are building a mobile-first work order application for traffic signal work orders. The app will be integrated with the Data Tracker, and will allow field technicians to quickly and accurately review and respond to traffic signal issues from a mobile device.

### Resources
- [Link to Drive folder](https://drive.google.com/drive/folders/1WTiIt-xGLAjQpNdGsGYGcilmkydV4MKk?usp=sharing)
- [Github](https://github.com/cityofaustin/atd-mobile-signals-work-orders)

**Comment by johnclary** [comment]
looking for a status update @mateoclarke @dianamartin. Billy has confirmed that the he will continue to onboard technicians to the app? There there are no immediate issues to address? I'll tap Jace to lead PM tasks going forward.

**Comment by dianamartin** [comment]
Update: `mobile-signals` web app shortcut has been updated to the new domain URL. I've add all technicians in user role that allows them access to use the mobile-signals application. Billy should be disseminating the new URL to the application.

Apps & Dev team will be on standby to troubleshoot anything that comes back from user implementation.

**Comment by JaceDeloney** [comment]
**Status Update**
We have not received any new requests related to the Mobile Signal Work Orders app.

**Comment by amenity** [comment]
@dianamartin pointed out that our extensive work on [[Feature] Enhanced AMD Work Order Management](https://github.com/cityofaustin/atd-data-tech/issues/5125#) will break this app. 

According to Jorge, it is used very little as most technicians are already working on their laptops in the field. Need to discuss with Billy & BC, but Diana and I recommend deprecating the app for the time being. @mateoclarke - any thoughts?

**Comment by mateoclarke** [comment]
Yeah, I logged into the analytics software (Clicky) we had installed to see if we have any active users but it seems like our free account got disabled because we hadn't logged in. So can't add much quantitative info about usage, but it seems like its been pretty low for awhile. Not against deprecating it if that's the best move. Not sure about the level of effort to refactor the existing app without having a better understanding of the changes.

**Comment by mddilley** [comment]
@ChristinaTremel Not a big priority at all but I wanted to let you know that I created an issue to archive the repo for this app and to clean up the dev resources associated with it - https://github.com/cityofaustin/atd-data-tech/issues/19248. We won't move forward with that w/o checking in with you first.

It came up since this app's url is kept secure by [this work](https://github.com/cityofaustin/atd-data-tech/issues/19214) along with Moped and Vision Zero Editor.

**Comment by mddilley** [comment]
@ChristinaTremel Should we make any updates here when we spin down this deployment in https://github.com/cityofaustin/atd-data-tech/issues/19248?

**Comment by dianamartin** [comment]
@mddilley should we have the old URLs gathered and any screen shots so we can reference this work for historical purposes or just reference that we've done this kind of work before?

**Comment by mddilley** [comment]
@dianamartin I think that the page is still a good representation of what work was done and where the code lives. https://austinmobility.io/products/251 

I think updating the **Type** and **Status** values on this issue could be valuable since they are showing as "New Application" and "Ongoing" on the DTS site.

I'm also on the hunt for an example of another app that was retired. We may not have tags that fit this situation.

**Comment by dianamartin** [comment]
@Dilley, ***@***.***> the vision zero in action app was retired. @Applewhite, ***@***.***> if this a good example?

Diana Martin, IT Supervisor Sr.
Data and Technology Services Division | DTS
Transportation and Public Works | TPW
City of Austin
________________________________

**Comment by amenity** [comment]
Missed that, @dianamartin ... yes, retiring! cc/ @ChristinaTremel 

Interestingly, Knack is pursuing a "BYO UI" offering as of a month ago.
